.travis.yml

```diff
@@ -34,6 +34,7 @@ env:
   global:
   - AWS_DEFAULT_REGION=us-east-1
   - DSS_S3_TEST_BUCKET=hca-dss-test
+  - DSS_S3_TEST_WRITE_BUCKET=hca-dss-test-write-dst
```

Comment:
Not sure if I agree with this nomenclature, I find this naming confusing. I would suggest naming a "test staging bucket" for testing transfers across buckets. Otherwise I think our main bucket (DSS_S3_TEST_BUCKET) should be used for both read and write testing.